### PR TITLE
[Testing:TAGrading] Add cypress test for Numeric gradeable

### LINKED ISSUE #8895: Write Cypress test cases for Numeric/Text gradeable
See comment on PR #8867 

"Given that this is a relatively low-traffic page which seems to get broken on a regular basis, it might be good to write some basic Cypress tests for this page. In particular, this action of navigating both the UI and the text within each of the cells seems prone to breaking in the future. It should be relatively straightforward to write a basic test for this."

### PR DESCRIPTION
**Pull request by charvimehradu**
### Please check if the PR fulfills these requirements:

* [x] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant
* [ ] Screenshots are attached to Github PR if visual/UI changes were made

### What is the current behavior?
Closes #8895 

Currently, there are no tests written for testing the functionality that double-clicking a cell activates typing mode and we can use arrow keys to control the cursor position within that cell.

### What is the new behavior?

Added cypress test to:
Test that in the sample numeric grading double click can activate typing mode.
Test that in the sample numeric grading double click can  simulate typing within a cell.
Test that in typing mode arrow keys can move the cursor within the cell.

### CODE REVIEW

**Comment by williamjallen** [comment]
Hi @charvimehradu, apologies for the delay in reviewing this.  I just ran the tests for you.  @IDzyre is our local Cypress expert, so I have requested his review.

File: site/cypress/e2e/Cypress-Gradeable/numeric_grading.spec.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+describe('Test that in the sample numeric grading test that', () => {
+  beforeEach(() => {
+    cy.visit(['sample', 'gradeable', 'grading_test', 'grading']);
+    cy.login('ta');
+  });
+
+  it('double click can activate typing mode', () => {
+    cy.get('#cell-1-0-0').dblclick();
+    cy.get('input#cell-1-0-0.option-small-box.cell-grade').should('exist');
```
Comment (IDzyre) [inline site/cypress/e2e/Cypress-Gradeable/numeric_grading.spec.js:9]: This should be a better selector, consider adding a cypress selector to the html element. https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

**Review by IDzyre — CHANGES_REQUESTED** [review]
After a selector is changed, and the js-linting is fixed i'll re-review.

**Review by IDzyre — APPROVED** [review]
Tests look alright to me, i am not familiar with the feature its self to know much about it.

**Comment by charvimehradu** [comment]
Thanks for testing it @IDzyre

**Pull request merged**